wagesAnalysis.sh

```diff
@@ -0,0 +1,31 @@
+### Script for getting information about highest and lowest earning people
+### Usage: bash wagesAnalysis.sh
+
+#
+echo "Highest earner:"
+### Gets the gender, yearsExperience, and wages of the highest earner
+cat wages.csv | sed 's/,/ /g' | sort -k 4 -n | tail -1 | cut -d " " -f1,2,4 
+
+#
+echo "Lowest earner:"
+### Gets the gender, yearsExperience, and wages of the lowest earner
+cat wages.csv | sed 's/,/ /g' | sort -k 4 -n | head -2 | grep -v "gender" | cut -d " " -f1,2,4 
+
+#
+echo "Number of females in the top 10 earners:"
+cat wages.csv | sed 's/,/ /g' | sort -k 4 -n | grep -v "gender" | head -10 | cut -d " " -f1,2,4 | grep "female"| wc -l 
```

Comment:
This gives the number of females in the bottom 10. Either use tail -10 or the -r flag in sort.
-0.25 pts